Sources/LLBuildManifest/Node.swift

```diff
@@ -37,15 +45,27 @@ public struct Node: Hashable, Codable {
         return String(self.name.dropFirst().dropLast())
     }
 
-    public static func virtual(_ name: String) -> Node {
+    public static func virtual(_ name: String, isCommandTimestamp: Bool = false) -> Node {
         precondition(name.first != "<" && name.last != ">", "<> will be inserted automatically")
-        return Node(name: "<" + name + ">", kind: .virtual)
+        return Node(
+            name: "<" + name + ">",
+            kind: .virtual,
+            attributes: isCommandTimestamp ? .init(isCommandTimestamp: isCommandTimestamp) : nil
+        )
     }
 
     public static func file(_ name: AbsolutePath) -> Node {
         Node(name: name.pathString, kind: .file)
     }
 
+    public static func file(_ name: AbsolutePath, isMutated: Bool) -> Node {
```

Comment:
Couldn't create a default parameter for this one without breaking `.map(Node.file)` constructs that we have elsewhere in our code.